cmd/car/testdata/script/index-create.txt

```diff
@@ -0,0 +1,3 @@
+car index create ${INPUTS}/sample-v1.car sample-v1.car.idx
+car detach-index ${INPUTS}/sample-wrapped-v2.car sample-wrapped-v2.car.idx
+cmp sample-v1.car.idx sample-wrapped-v2.car.idx

```

Comment:
lmk if this is too simple and I can either add an index to the inputs to compare against or add something like:

```
car detach-index list sample-v1.car.idx
cmp stdout index-list.txt

-- index-list.txt --
...
```

Reply:
no, I think this is fine; we're not exactly doing sophisticated and high-coverage tests here in cmd

this does make me think though that maybe `detach` should be moved to a subcmd of `index` too

Reply:
> this does make me think though that maybe detach should be moved to a subcmd of index too

I agree, but figured I'd let the maintainers make that call and if they'd be ok with a breaking move or want both supported.

Reply:
we don't exactly have an SLA on this, cmd is currently just a grab-bag of utils we collectively find useful

Reply:
SGTM. Happy to submit a PR to move it once this one is closed out